context/entrypoint.sh

```diff
@@ -23,6 +23,14 @@ if [ "$EXTRA_PIP_PACKAGES" ]; then
     timeout ${PIP_TIMEOUT:-600} pip install $EXTRA_PIP_PACKAGES || exit $?
 fi
 
+if [ "$(uname -m)" = "aarch64" ]; then
+    # Check if the CUDA version is 12.8
+    if nvcc --version | grep -q "CUDA 12.8"; then
```

Comment:
I don't think this works. Could do something like `nvcc --version | grep "Cuda" | grep "12.8"` maybe.

But, `CUDA_VERSION` is available, so this might be better:
```suggestion
    if [[ "$CUDA_VERSION" = 12.8* ]]; then
```